doc-content/shared-kie-docs/src/main/asciidoc/Commands/runtime-commands-marshalling-ref.adoc

```diff
@@ -0,0 +1,88 @@
+[id='runtime-commands-marshalling-ref_{context}']
+= Runtime commands marshalling
+
+*<@Maciej/@Tiho and @Mario/@Matteo: Is this section complete and correct? Any other marshalling requirements? Maciej pointed me to this https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/model/MarshallingRoundTripTest.java[MarshallingRoundTripTest] for example, but I'm not sure what if anything is required on the user's part.>*
+
+Marshalling and unmarshalling of the {PRODUCT} rule services commands requires the following classes, depending on the marshaller type:
+
+XStream::
+For the XStream commands marshaller, use the `DroolsHelperProvider` to obtain an `XStream` instance where the commands converters are registered. Also ensure that the `drools-compiler` library is present on the class path.
++
+--
+[source,java]
+----
+BatchExecutionHelper.newXStreamMarshaller().toXML(command);
+----
+
+[source,java]
+----
+BatchExecutionHelper.newXStreamMarshaller().fromXML(xml);
+----
+
+The fully-qualified class name of the `BatchExecutionHelper` class is `org.kie.internal.runtime.helper.BatchExecutionHelper`.
+--
+
+JSON::
+For the JSON commands marshaller, use the `DroolsHelperProvider` to obtain a `JSON` instance where the commands converters are registered. Also ensure that the `drools-compiler` library is present on the class path.
++
+--
+[source,java]
+----
+BatchExecutionHelper.newJSonMarshaller().toXML(command);
```

Comment:
same here, I would remove completely this section - meaning this file

Reply:
@mswiderski Please see my response/questions in the [JIRA](https://issues.jboss.org/browse/BXMSDOC-3364).